docker-entrypoint.sh

```diff
@@ -22,6 +22,27 @@ args=("$@")
 
 cd /flink-kubernetes-operator || exit
 
+maybe_enable_jemalloc() {
+    if [ "${DISABLE_JEMALLOC:-false}" == "false" ]; then
+        JEMALLOC_PATH="/usr/lib/$(uname -m)-linux-gnu/libjemalloc.so"
+        JEMALLOC_FALLBACK="/usr/lib/x86_64-linux-gnu/libjemalloc.so"
```

Comment:
Why do we fallback and not fail fast?

Reply:
Refer to the issue [FLINK-28057](https://issues.apache.org/jira/browse/FLINK-28057),

Maybe the user base image cannot use or install jemalloc，so will try fallback.